documentation/decisions/adr/2023-09-07-data-replication-tool.md

```diff
@@ -46,19 +55,184 @@ Additionally, [AWS DMS is FedRAMP compliant](https://aws.amazon.com/compliance/s
+
+Non AWS solutions require traffic to leave the AWS network and traverse the public internet. While that traffic can be encrypted with a VPN, that is inherently less secure than keeping the traffic within the AWS boundary.
+
+## Implementation Guide
+
+After reviewing the AWS documentation on configuring DMS where the target and source database reside in different VPCs, the recommended approach is to use AWS DMS with VPC Peering to permit the necessary traffic. The following details the division of tasks between Nava and MicroHealth to set up the necessary services.
+
+VPC Peering must be configured before DMS can complete, however in order to limit VPC Peering traffic to only DMS, the DMS instance and subnet must be created with its CIDR block and IP address to share with MicroHealth for updating their route tables.
+
+#### VPC Peering
+
+##### Nava
+- Confirm there are no overlapping IPv4 or IPv6 CIDR blocks
+  - if there are overlapping CIDR blocks create a new VPC with non overlapping CIDR blocks and migrate resources
+- Info to share with MicroHealth
+  - DMS security group ID
+  - DMS instance subnet CIDR block
+  - DMS instance IP address
+- [Create vpc peering connection](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html)
+- [Update your security groups to reference peer security groups](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-security-groups.html)
+- - [Update route tables for peering connection to the db subnet](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html) or to a [specific IP address](https://docs.aws.amazon.com/vpc/latest/peering/peering-configurations-partial-access.html)
+
+
+##### MicroHealth
+- Confirm there are no overlapping IPv4 or IPv6 CIDR blocks
+- Share information with Nava:
+  - Region information for the VPC
+  - AZ info for the DB
+  - Account ID
+  - VPC ID
+  - Database or new data replication security group ID
+- [Accept VPC Peering connection request](https://docs.aws.amazon.com/vpc/latest/peering/accept-vpc-peering-connection.html)
+- [Update route tables for peering connection to the db subnet](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html)
+- [Update route tables for peering connection to the db subnet](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html) or to a [specific IP address](https://docs.aws.amazon.com/vpc/latest/peering/peering-configurations-partial-access.html)
+
+
+#### AWS DMS Service
+
+##### Nava
+- Create a user with AWS Identity and Access Management (IAM) credentials that allows you to launch Amazon RDS and AWS Database Migration Service (AWS DMS) instances in your AWS Region.
+- Size your target PostgreSQL database host based on the current db host load profile.
+- Create the schemas in the target database
+- Create the AWS DMS user to connect to your target database:
+  - ```
+    CREATE USER postgresql_dms_user WITH PASSWORD 'password';
```

Comment:
can we note this as <password> instead so that it's clear we'll replace it with an actual password (and not one shown on our open GitHub repo)?